src/writer.f90

```diff
@@ -15,7 +15,7 @@ module writer_mod
     end interface raw_write
 
     private
-    public :: normalise_fluence, write_data, write_detected_photons
+    public :: normalise_fluence, write_phase, write_detected_photons
```

Comment:
this is better off as write_data as it more generic. Thiough it would be a good idea to have a complex write_data. Though not sure how the nrrd file format would handle this